.gitignore

```diff
@@ -1,4 +1,4 @@
-vendor
+/vendor/
```

Comment:
Not really. Currently, it is ignoring everything with `vendor` in it. Since I configured bower to install packages in `Resources/public/js/vendor`, it was ignoring that too (which I don't want).